sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -70,6 +70,7 @@
   "dependencies": {
     "@azure/core-arm": "1.0.0-preview.3",
     "@azure/core-http": "1.0.0-preview.3",
+    "@azure/core-tracing": "1.0.0-preview.2",
```

Comment:
since you are now using the re-exported tracing types from core-http, I don't believe we need to add `core-tracing` as a dependency here